### ISSUE TITLE
Checker Framework Eclipse plugin v1.5.0 does not work with Juno

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#198]
```
What steps will reproduce the problem?
1. Install Eclipse Juno Service Release 1
2. Install Checker Framework Eclipse plugin v1.5.0
3. Attempt to run the Checker Framework on a Java project.

I was previously using version 1.1.4 of the Checker Framework. This version 
appeared to have an automatic builder (checkers.eclipse.checkerbuilder) that 
would execute the Checker Framework each time a file has been changed. That 
builder appears to have been removed from newer versions of the Checker 
Framework, which is very unfortunate as that feature is very useful.

Since the automatic builder has been removed, I attempted to right-click my 
Java project and select Checker Framework > Run Built-in Checker > Nullness 
Checker. This apparently does nothing. There is nothing in the Checkers Plugin 
Console and no error in the Eclipse Error Log. The nullness checker does appear 
to get run as no nullness problems in the code are marked (I know there are 
cases where code should be flagged). The same behavior occurred if I 
right-clicked the project and selected Checker Framework > Run Selected 
Checkers.

I then attempted to create a new Java project as the previous project was still 
configured to use the non-existent checkers.eclipse.checkerbuilder builder. 
This time, I would get the following stack trace in the Eclipse Log whenever I 
clicked Checker Framework > Run Built-in Checker > Nullness Checker or Checker 
Framework > Run Selected Checkers:

java.lang.IllegalArgumentException: Path must include project and resource 
name: /some.test.project
    at org.eclipse.core.runtime.Assert.isLegal(Assert.java:63)
    at org.eclipse.core.internal.resources.Workspace.newResource(Workspace.java:2169)
    at org.eclipse.core.internal.resources.Container.getFile(Container.java:208)
    at checkers.eclipse.util.ResourceUtils.outputLocation(ResourceUtils.java:276)
    at checkers.eclipse.actions.CheckerWorker.pathOf(CheckerWorker.java:157)
    at checkers.eclipse.actions.CheckerWorker.classPathEntries(CheckerWorker.java:220)
    at checkers.eclipse.actions.CheckerWorker.classPathOf(CheckerWorker.java:208)
    at checkers.eclipse.actions.CheckerWorker.runChecker(CheckerWorker.java:116)
    at checkers.eclipse.actions.CheckerWorker.work(CheckerWorker.java:105)
    at checkers.eclipse.actions.CheckerWorker.run(CheckerWorker.java:83)
    at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

So there appears to be no way to get the Eclipse plugin v1.5.0 to work with 
Juno. I am using Java 1.7.0_09 in case that matters.
```

Original issue reported on code.google.com by `innatedi...@gmail.com` on 25 Jan 2013 at 2:03
